compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/Clickable.kt

```diff
@@ -693,7 +693,7 @@ internal inline fun Modifier.clickableWithIndicationIfNeeded(
             // Fast path - indication is managed internally
             indication is IndicationNodeFactory -> createClickable(
                 interactionSource,
-                platformIndication(indication) as IndicationNodeFactory
+                indication
```

Comment:
Did you test that the focus indication is still visible after user presses the Tab for `Button` (clickable) or for `CheckBox` (toggleable)?